### DISCUSSION TITLE
Font ligatures with Fira Code and Code Overview (Linux)

### DISCUSSION THREAD

**Discussion by Hellvol** [consulo/consulo#507] [Q&A]
Hello there. First, I would like to point out that I love Consulo and you are great.
I have two minor questions: How can I enable font ligatures? I'm using FiraCode font, and it supports ligatures, I have the font set in Consulo Settings and the "Enable font ligatures" checkbox marked, but it still doesn't work, as you can see in the "==" inside my if condition.

![Screenshot_20211210_203522](https://user-images.githubusercontent.com/62046614/145654077-f353cd63-05d4-4628-a14d-4f78761a651c.png)

The other question is if it is possible to have a "Code Overview" instead of a scrollbar, like it happens in Visual Studio Code:

![Screenshot_20211210_204410](https://user-images.githubusercontent.com/62046614/145654574-0a8ec6fd-ae71-4a3e-8da9-aded03c88612.png)

You can see that FiraCode with font ligatures works in my Visual Studio Code.

Thank you for any reply and for your amazing work.

**Comment by VISTALL** [comment]
Hello. About code overview, not supported (https://github.com/consulo/consulo/issues/423)

about ligatures. They must works as expected. Can you switch to color scheme -> Consulo Light (it's light, source code pro + ligatues), and check supported they or not.

Can you attach info from About? (help -> about)

**Comment by yoddaa** [comment]
== ===
